src/chapters/inheritance/cat-classes.rst

```diff
@@ -230,11 +231,12 @@ In a previous lesson, we introduced the :ref:`special-methods` ``Equals`` and
 ``ToString``. All classes contain default implementations of these methods that 
 can be overridden.
 
-In fact, these default methods are part of a class called ``Object``. If
-a class does not explicitly extend another class, then it implicitly
-extends ``Object``. So the default implementations of ``Equals`` and
-``ToString`` (along with a few `other
-methods <https://docs.microsoft.com/en-us/dotnet/api/system.object?view=netframework-4.8>`__)
+In fact, these default methods are part of a class called ``Object``.
+If a class does not explicitly extend another class, then it implicitly extends ``Object``.
+In the case of ``Cat`` and ``HouseCat``, ``HouseCat`` explicitly extends ``Cat`` and ``Cat`` implicitly extends ``Object``.
+This means that both ``Cat`` and ``HouseCat`` can access different methods and members of the ``Object`` class.
```

Comment:
yeah this is helpful! I also just don't like this sentence: "If a class does not explicitly extend another class, then it implicitly extends ``Object``." 
Because all classes implicitly inherit from Object - whether or not they also explicitly inherit another class.
I'd just ditch that sentence and edit the following one to something like:
"In the case of ``Cat`` and ``HouseCat``, ``HouseCat`` explicitly extends ``Cat`` and both ``HouseCat`` and ``Cat`` implicitly extend ``Object``."